ceph-iscsi.spec

```diff
@@ -37,6 +37,7 @@ BuildArch:      noarch
 Obsoletes:      ceph-iscsi-config
 Obsoletes:      ceph-iscsi-cli
 
+Requires:       tcmu-runner >= 1.3.0
```

Comment:
Nit: probably ```>= 1.4.0``` now

Reply:
Fixed. I've also created a PR to update the ceph documentation accordingly - https://github.com/ceph/ceph/pull/28494